src/dockerize.ts

```diff
@@ -52,14 +54,17 @@ export function dockerize(realHandler: YargsHandler): YargsHandler {
 				'-v', `${CY_PROJECT_PATH}:/e2e`,
 				'-w', '/e2e',
 				'-e', 'HOME=/opt/cypress/home',
+				...cypressEnvVars,
 				'-e', 'CYPRESS_CACHE_FOLDER=/opt/cypress/cache',
-				'-e', `CYPRESS_BASE_URL=${process.env.CYPRESS_BASE_URL}`,
 				'-e', 'HTTP_PROXY', '-e', 'HTTPS_PROXY', '-e', 'NO_PROXY',
 				'--entrypoint=bash',
 				'-t',
 				CY_DOCKER_IMAGE,
-				'-c',
+				'-c', // bash command execution flag
 				[
+					// create group optionally
+					`groupadd -g ${groupId} testergroup || true`,
```

Comment:
Cant we just pass the executor group and user?

Reply:
no, because some commands during cypress reporting complain about non-existing user